website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -4,391 +4,275 @@ linkTitle: Releasing
-ex. `v1.3.x` to `https://v1-3-x.sdk.operatorframework.io`. You can ping SDK [approvers][doc-owners] to ensure a
-[release branch](#release-branches) is created prior to the release and that this mapping is created.
-
-### 3. Create release branch for Netlify
-
-The release branch must be created before the release occurs to appease the Netlify website configuration demons.
-You can do so by running the following before proceeding with the release, assuming the upstream SDK is the `upstream` remote repo:
+Create a new branch from the release branch, which should already exist for the desired minor version,
+to push the release commit to:
 
 ```sh
-$ git checkout master
-$ git pull
-$ git checkout -b v1.3.x
-$ git push -u upstream v1.3.x
+git checkout v1.3.x
+git pull
+git checkout -b release-v1.3.1
 ```
 
-### 4. Create a PR for release version, CHANGELOG.md, and migration guide updates
-
-Once all PR's needed for a release have been merged, branch from `master`:
+Run the pre-release `make` target:
 
 ```sh
-$ git checkout master
-$ git pull
+make prerelease RELEASE_VERSION=v1.3.1
 ```
 
-If making a patch release, check out the corresponding minor version branch:
+The following changes should be present:
 
-```sh
-$ git checkout v1.2.x
-$ git pull
-```
+- `changelog/generated/v1.3.0.md`: commit changes (created by changelog generation).
+- `changelog/fragments/*`: commit deleted fragment files (deleted by changelog generation).
 
-Create a new branch to push release commits:
+Commit these changes and push:
 
 ```sh
-$ git checkout -b release-v1.3.0
+git add --all
+git commit -m "Release v1.3.1"
+git push -u origin release-v1.3.1
 ```
 
-Run the CHANGELOG and migration guide generator:
+### 2. Lock down branch `v1.3.x`
 
-```sh
-$ GEN_CHANGELOG_TAG=v1.3.0 make changelog
-```
+Lock down the `v1.3.x` branch to prevent further commits before the release completes.
+See [this section](#locking-down-branches) for steps to do so.
 
-Commit the following changes:
+### 3. Create and merge a new PR
 
-- `website/content/en/docs/installation/install-operator-sdk.md`: update the linux and macOS URLs to point to the new release URLs.
-- `CHANGELOG.md`: commit changes (updated by changelog generation).
-- `website/content/en/docs/upgrading-sdk-version/v1.3.0.md`: commit changes (created by changelog generation).
-- `changelog/fragments/*`: commit deleted fragment files (deleted by changelog generation).
-- **(Major and minor releases only)** `website/config.toml`: update `version_menu = "Releases"` with the patch-less version string `version_menu = "v1.3"`,
-and add the following lines under `[[params.versions]]` for `master`:
-  ```toml
-  [[params.versions]]
-    version = "v1.3"
-    url = "https://v1-3-x.sdk.operatorframework.io"
-  ```
+Create and merge a new PR for the above commit. You can force-merge your PR to the locked-down `v1.3.x`
+if you have admin access to the operator-sdk repo, or ask an administrator to do so.
+
```

Comment:
Rephrase: Create and merge a new PR for the commit created in [Step 1](#1-create-and-push-a-release-commit-1). You can ....